<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="sony-PC" name="Sample.TestCase1" tests="12" failures="0" timestamp="25 Sep 2021 06:06:20 GMT" time="1.482" errors="1">
  <testcase name="function1" time="0.000" classname="Sample.TestCase1"/>
  <testcase name="function2" time="1.466" classname="Sample.TestCase1">
    <error type="org.testng.internal.thread.ThreadTimeoutException" message="Method Sample.TestCase1.function2() didn&#039;t finish within the time-out 1000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method Sample.TestCase1.function2() didn't finish within the time-out 1000
]]>
    </error>
  </testcase> <!-- function2 -->
  <testcase name="function1" time="0.000" classname="Sample.TestCase1"/>
  <testcase name="function1" time="0.000" classname="Sample.TestCase1"/>
  <testcase name="function1" time="0.000" classname="Sample.TestCase1"/>
  <testcase name="function1" time="0.000" classname="Sample.TestCase1"/>
  <testcase name="function1" time="0.016" classname="Sample.TestCase1"/>
  <testcase name="function1" time="0.000" classname="Sample.TestCase1"/>
  <testcase name="function3" time="0.000" classname="Sample.TestCase1"/>
  <testcase name="function1" time="0.000" classname="Sample.TestCase1"/>
  <testcase name="function1" time="0.000" classname="Sample.TestCase1"/>
  <testcase name="function1" time="0.000" classname="Sample.TestCase1"/>
</testsuite> <!-- Sample.TestCase1 -->
